simulariumio/data_objects/agent_data.py

```diff
@@ -197,6 +212,27 @@ def get_type_names(
                 )
         return result
 
+    @staticmethod
+    def get_display_data(type_mapping: Dict[str, Any]) -> Dict[str, DisplayData]:
```

Comment:
is there a well defined type for the values in type_mapping?

Reply:
No, it's coming from a JSON file that was read back in. The keys are strings, but the values are strings, numbers, or another dict, depending on if each agent type has geometry info. I added to the function comment to clarify